app/home/directives/home-series.component.ts

```diff
@@ -0,0 +1,106 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {Observable} from 'rxjs';
+
+import {HalDoc} from '../../shared/cms/haldoc';
+import {StoryModel} from '../../storyedit/models/story.model';
+import {SpinnerComponent} from '../../shared/spinner/spinner.component';
+import {ImageLoaderComponent} from '../../shared/image/image-loader.component';
+import {TimeAgoPipe} from '../../shared/date/timeago.pipe';
+import {HomeStoryComponent} from './home-story.component';
+
+@Component({
+  directives: [SpinnerComponent, ImageLoaderComponent, HomeStoryComponent],
+  pipes: [TimeAgoPipe],
+  selector: 'home-series',
+  styleUrls: ['app/home/directives/home-series.component.css'],
+  template: `
+    <header *ngIf="noseries">
```

Comment:
Yep - it does.  This poorly named component displays a list of stories.  The list either has a parent-series or does not (via that `noseries` input).  In the HomeComponent, you'll see a switch that decides how many of these to display (and how many rows of stories) based on if you have a series or not.